telegram-bot/main.go

```diff
@@ -168,12 +192,15 @@ func main() {
 	}
 
 	fmt.Println("Connected to MongoDB!")
+	
+	tweet()
```

Comment:
Make this function concurrent so that it doesn't block the execution. 

https://tour.golang.org/concurrency/1 may help.

As pointed out by @aayushmau5